### PR TITLE
spandsp 0.0.6 (new formula)

### PR DESCRIPTION
**Pull request by afh**
DSP functions library for telephony

### CODE REVIEW

**Comment by dunn** [comment]
Probably just needs `libtiff`:
```
checking tiffio.h usability... no
checking tiffio.h presence... no
checking for tiffio.h... no
checking for TIFFOpen in -ltiff... no
configure: error: "Cannot build without libtiff (does your system require a libtiff-devel package?)"
```

File: Library/Formula/spandsp.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Spandsp < Formula
+  desc "DSP functions library for telephony"
+  homepage "http://www.soft-switch.org"
+  url "http://www.soft-switch.org/downloads/spandsp/spandsp-0.0.6.tar.gz"
+  sha256 "cc053ac67e8ac4bb992f258fd94f275a7872df959f6a87763965feabfdcc9465"
+
+  depends_on "libtiff"
+
+  def install
+    ENV.deparallelize
```
Comment (bfontaine) [inline Library/Formula/spandsp.rb]: Is upstream aware of this?

**Comment by apjanke** [comment]
This test failure on 10.11 looks like a problem with our test bot Rubocop setup, not with the formula.

```
==> brew audit spandsp --strict --online
==> brew style spandsp
ERROR:  Error installing rubocop:
	rainbow requires Ruby version >= 1.9.2.
Successfully installed ast-2.1.0
Successfully installed parser-2.2.3.0
Successfully installed powerpack-0.1.1
Successfully installed ruby-progressbar-1.7.5
Error: Failure while executing: gem install --no-ri --no-rdoc --user-install rubocop -v 0.35.1
==> FAILED
```

File: Library/Formula/spandsp.rb (hunk 1 above)
Comment (apjanke) [inline Library/Formula/spandsp.rb]: There's no public bug tracker for spandsp. The spandsp mailing list looks like the only public forum.

http://lists.soft-switch.org/mailman/listinfo
http://lists.soft-switch.org/pipermail/spandsp/

The archives are not searchable; might be best to just fire off a new bug report message.

**Comment by DomT4** [comment]
We've actually fixed that problem recently, but I believe Travis will be building from when the commit was pushed some three weeks ago.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Decided to have a crash yesterday. Trying again.

**Comment by apjanke** [comment]
Went and merged this since Jenkins passed, and the one failure is known to be an unrelated test bot issue and the audit actually passes.

Thank you for your contribution to Homebrew!

**Pull request closed without merging**